gramps/plugins/graph/gvfamilylines.py

```diff
@@ -809,9 +819,16 @@ def write_people(self):
                     media = self._db.get_media_from_handle(media_handle)
                     media_mime_type = media.get_mime_type()
                     if media_mime_type[0:5] == "image":
-                        image_path = get_thumbnail_path(
-                            media_path_full(self._db, media.get_path()),
-                            rectangle=media_list[0].get_rectangle())
+                        if self._imagesize == SIZE_NORMAL:
```

Comment:
The `if` statement is not necessary here.  Use `size=self._imagesize` in the call to `get_thumbnail_path`.